pkg/account/import.go

```diff
@@ -13,17 +15,32 @@ import (
 )
 
 var (
-	NotAbsPath = errors.New("keypath is not absolute path")
+	// ErrNotAbsPath when keypath not absolute path
+	ErrNotAbsPath   = errors.New("keypath is not absolute path")
+	ErrBadKeyLength = errors.New("Invalid private key (wrong length)")
 )
 
-func ImportKeyStore(keypath, passphrase string) (string, error) {
-	if !path.IsAbs(keypath) {
-		return "", NotAbsPath
+// ImportFromPrivateKey allows import of an ECDSA private key
+func ImportFromPrivateKey(privateKey, name, passphrase string) (string, error) {
+	if name == "" {
+		name = generateName() + "-imported"
 	}
-	keyJSON, readError := ioutil.ReadFile(keypath)
-	if readError != nil {
-		return "", readError
+	privateKeyBytes, err := hex.DecodeString(privateKey)
+	if err != nil {
+		return "", err
+	}
+	if len(privateKeyBytes) != 32 {
```

Comment:
magic number 32 should be const, put in `pkg/common/values.go`

Reply:
done